app/controllers/distributions_by_county_controller.rb

```diff
@@ -0,0 +1,40 @@
+class DistributionsByCountyController < ApplicationController
+  include DateRangeHelper
+  include DistributionHelper
+
+  def show
+    setup_date_range_picker
+    distributions = current_organization.distributions.includes(:partner).during(helpers.selected_range)
+    breakdown_hash = {}
```

Comment:
This method is too long to be in a controller. It should be in a service class. I'd also recommend not using a hash at all here, but creating a strongly typed Struct to store these values.

Reply:
Moving it to a service in the next push.  Considering the advice re the struct.  Will probably do that too, because my tests for the service revealed a whole lotta awkwardness.